### PR TITLE
feat!: add support for BGP keepalive_interval

### PR DESCRIPTION
**Pull request by jmymy**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by jmymy** [comment]
@morgante @bharathkkb 👋🏼

**Comment by jmymy** [comment]
@morgante @bharathkkb 👋🏼

**Comment by hdhruna** [comment]
This change is required if anyone uses latest releases of `cloud-nat` and `cloud-router` modules. @morgante @bharathkkb please merge

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR @jmymy and sorry for the delay.

File: versions.tf [hunk 1]
```diff
@@ -20,7 +20,7 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 4.13, < 5.0"
```
[the remark this answers is not in the document]
Comment (jmymy) [reply versions.tf:23]: that was the first version to support keep alive params

**Pull request merged**